mne/io/neuralynx/neuralynx.py

```diff
@@ -43,25 +43,29 @@ def read_raw_neuralynx(
     Notes
     -----
     Neuralynx files are read from disk using the Neo package (http://neuralensemble.org/neo/).
-    Currently, only reading of the .ncs files is supported. ``raw.info["meas_date"]``
-    is determined based on the ``recording_opened`` property of the first .ncs file
-    in the dataset.
+    Currently, only reading of the ``.ncs files`` is supported.
 
-    Channel-specific high and lowpass frequencies are determined based on the
-    ``DspLowCutFrequency`` and ``DspHighCutFrequency`` header fields, respectively.
-    If no filters were used for a channel, the default lowpass is set to the Nyquist
-    frequency and the default highpass is set to 0. ``raw.info["highpass"]`` and
-    ``raw.info["lowpass"]`` are then set to the maximum and minimum values across
-    all .ncs sfiles, respectively.
+    ``raw.info["meas_date"]`` is read from the ``recording_opened`` property
+    of the first ``.ncs`` file (i.e. channel) in the dataset (a warning is issued
+    if files have different dates of acquisition).
 
-    Other header variables can be inspected using neo directly. For example:
+    Channel-specific high and lowpass frequencies of online filters are determined
+    based on the ``DspLowCutFrequency`` and ``DspHighCutFrequency`` header fields,
+    respectively. If no filters were used for a channel, the default lowpass is set
+    to the Nyquist frequency and the default highpass is set to 0.
+    If channels have different high/low cutoffs, ``raw.info["highpass"]`` and
+    ``raw.info["lowpass"]`` are then set to the maximum highpass and minimumlowpass
+    values across channels, respectively.
+
+    Other header variables can be inspected using Neo directly. For example:
 
     .. code-block:: python
     from neo.io import NeuralynxIO
     fname = 'path/to/your/data'
     nlx_reader = NeuralynxIO(dirname=fname)
     print(nlx_reader.header)
     print(nlx_reader.file_headers.items())
+
```

Comment:
@larsoner mind having a look at this docstring in `read_raw_neuralynx()`? I'm formatting something wrong, but can't figure out what. Ruff doesn't detect anything locally.

With this commit, it's failing like so:
```
FAILED mne/tests/test_docstring_parameters.py::test_docstring_parameters - AssertionError: 2 errors found:
mne.io : mne.io.neuralynx.neuralynx.read_raw_neuralynx : GL02 : Closing quotes should be placed in the line after the last text in the docstring (do not close the quotes in the same line as the text, or leave a blank line between the last text and the quotes)
mne.io : mne.io.neuralynx.neuralynx.read_raw_neuralynx : GL03 : Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
```


But if I delete the blank line (l. 68), then I think sphinx will break when building docs and I see this log (full [CircleCI link](https://app.circleci.com/pipelines/github/mne-tools/mne-python/23176/workflows/20590454-077c-4384-a4db-2b3e181edd2d/jobs/64203)):
```
/home/circleci/project/mne/io/neuralynx/neuralynx.py:docstring of mne.io.neuralynx.neuralynx.read_raw_neuralynx:64: WARNING: Explicit markup ends without a blank line; unexpected unindent.

Reply:
Ah, wait, I think I see it. I guess I need an indent the code chunk after `.. code-block:: python`